src/components/MarkdownViewer/index.tsx

```diff
@@ -16,42 +16,41 @@
 import React, { FunctionComponent } from 'react';
 import gfm from 'remark-gfm';
 import frontmatter from 'remark-frontmatter';
-import ReactMarkdown, { ReactMarkdownPropsBase } from 'react-markdown';
+import ReactMarkdown, { ReactMarkdownOptions } from 'react-markdown';
 import { Prism as SyntaxHighlighter } from 'react-syntax-highlighter';
 import { tomorrow } from 'react-syntax-highlighter/dist/cjs/styles/prism';
 import Heading from '../Heading';
 import Text from '../Text';
 import Link from '../Link';
 
-interface RenderProps {
-    language: string;
-    value: string;
-}
-export interface MarkdownViewerProps extends ReactMarkdownPropsBase {
-    children?: any;
-}
-
-const renderers = {
-    code: ({ language, value }: RenderProps) => {
-        if (language && value) return <SyntaxHighlighter style={tomorrow} language={language} children={value} />;
-        else return <>{value}</>;
-    },
-    heading: (props: any) => {
-        const variant = `h${props.level}` as 'h1' | 'h2' | 'h3' | 'h4' | 'h5';
-        return <Heading variant={variant}>{props.children}</Heading>;
-    },
-    paragraph: (props: any) => {
-        return <Text variant="p">{props.children}</Text>;
-    },
-    link: (props: any) => {
-        return <Link href={props.href}>{props.children}</Link>;
+const components: ReactMarkdownOptions['components'] = {
+    code: ({ node, inline, className, children, ...props }) => {
+        const match = /language-(\w+)/.exec(className || '');
+        return !inline && match ? (
+            <SyntaxHighlighter
+                style={tomorrow}
+                language={match[1]}
+                children={String(children).replace(/\n$/, '')}
+                {...props}
+            />
+        ) : (
+            <code className={className} {...props} />
```

Comment:
This fallback one seems like being rendered as empty node:

`npm run build`

are rendered as 

```<p class="MuiTypography-root MuiTypography-body1"><code></code></p>```